rt_dashboard/javascript/src/components/tasks/tasks.js

```diff
@@ -112,8 +108,11 @@ export default class Tasks extends HTMLElement {
         appendNoDataRow(tbody, 'No jobs.', 3)
         return
       }
-
-      mapDataToElements(tbody, jobs, this.mapToRow)
+      const tbodyTemplate = createElement('tbody')
+      mapDataToElements(tbodyTemplate, jobs, this.mapToRow)
+      const table = this.shadowRoot.querySelector('table')
+      table.appendChild(tbodyTemplate)
+      table.removeChild(tbody)
```

Comment:
This should be quicker
```suggestion
      tbody.innerHTML = tbodyTemplate.innerHTML
```